grammars/hrm.cson

```diff
@@ -3,20 +3,20 @@
 'fileTypes': [
   'hrm'
 ]
-'firstLineMatch': '^-- HUMAN RESOURCE MACHINE PROGRAM --$'
+'firstLineMatch': '^ *-- HUMAN RESOURCE MACHINE PROGRAM --'
 'patterns': [
   {
-    'match': '--.*$'
+    'match': ' *--.*'
```

Comment:
Well, the change isn't incorrect. Though, I may have been overzealous in adding that particular asterisk all over the place :) 'COPYFROM 0-- test' does work in--game.